README.md

```diff
@@ -1,55 +1,55 @@
 # Dash Bio
 
-[![CircleCI](https://circleci.com/gh/plotly/dash-bio/tree/master.svg?style=svg)](https://circleci.com/gh/plotly/dash-bio)
+Dash Bio is a suite of bioinformatics components built to work with
+[Dash](https://github.com/plotly/dash/).
 
-🚧 *Work-in-Progress* 🏗️
+Demo:
+[https://dash-bio.plotly.host/Portal/](https://dash-bio.plotly.host/Portal/)
 
-Open-source bioinformatics components for Dash
+Documentation:
+[https://dash.plot.ly/dash-bio](https://dash.plot.ly/dash-bio)
 
-Demo: https://dash-bio.plotly.host/Portal/
+## Components
 
-## Progress
+The Dash Bio components each fall into one of three categories:
 
-#### Bioinformatics charts
+- Custom chart types
+- Sequence analysis tools
+- Visualization tools
 
-- **Dash Circos**
-- **Dash Clustergram**
-- Dash Phylocanvas
-- **Dash Cytoscape**
-- **Dash Oncoprint**
-- **Dash Needle**
-- **Dash Manhattan**
-- **Dash Volcano**
 
-#### Sequence analysis tools
+### Custom chart types
 
-- **Dash neXtProt Sequence Viewer**
-- Dash Multiple Sequence Alignment
-- Dash Track Viewer
-- **Dash Ideogram**
+- Dash Circos
+- Dash Clustergram
+- Dash Needle Plot
+- Dash Manhattan Plot
+- Dash Volcano Plot
 
-##### Molecular and structural bioinformatics visualization
+### Sequence analysis tools
+
+- Dash Alignment Viewer
+- Dash Sequence Viewer
+- Dash Onco Print
+
+### Visualization tools
 
 - Dash MolJS
 - Dash Speck
 
-## About
-
-Go to this link to learn about [Dash](https://plot.ly/products/dash/).
-Go to this link to learn about [Dash Bio](https://dash.bio).
-
-## Getting started
-
-1. Set up your Python environment:
-   ```bash
-   # Create a virtualenv
-   $ python3 -m venv venv
-   # Activate the virtualenv
-   $ . venv/bin/activate
-   # (On Windows, the above would be: venv\scripts\activate)
-   # Install the dev dependencies
-   $ pip install -r requirements.txt
-   ```
-2. Run `python index.py`
-3. Visit http://localhost:8050 in your web browser and click on the component
-   you want to explore.
+## Dash
+
+Learn more about Dash at
+[https://plot.ly/products/dash/](https://plot.ly/products/dash/).
+
+## Consulting and OEM
```

Comment:
What's OEM?

Reply:
https://en.wikipedia.org/wiki/Original_equipment_manufacturer
https://www.lifewire.com/what-is-oem-software-1701736

Referred to on the Plotly site here:
https://plot.ly/products/consulting-and-oem/

Reply:
Ok, thanks, "Consulting and OEM" makes sense then ("inquire about an OEM" less so).